docs/core/fees.md

```diff
@@ -0,0 +1,452 @@
+---
+title: Fees on Solana
+description:
+  "Your guide to transaction fees on Solana -- small fees paid to process
+  instructions on the network, based on computation and an optional
+  prioritization fee."
+keywords:
+  - instruction fee
+  - processing fee
+  - storage fee
+  - gas
+  - gwei
+altRoutes:
+  - /docs/intro/rent
+  - /docs/core/rent
+  - /docs/intro/transaction_fees
+  - /docs/intro/transaction-fees
+---
+
+The Solana blockchain has a few different types of fees and costs that are
+incurred to use the permissionless network. These can be segmented into a few
+specific types:
+
+- transaction fees
+- prioritization fees
+- rent (fee for on-chain data storage)
+
+## Transaction fees
+
+The small fee paid to process logic (instruction) within an on-chain program on
+the Solana blockchain is known as a "_transaction fee_".
+
+As each [transaction](/docs/core/transactions.md#transaction) (which contains
+one or more [instructions](/docs/core/transactions.md#instruction)) is sent
+through the network, it gets processed by the current validator leader. Once
+confirmed as a global state transaction, this _transaction fee_ is paid to the
+network to help support the economic design of the Solana blockchain.
+
+> Transaction fees are different from account data storage fee of [rent](#rent).
+> While transaction fees are paid to process instructions on the Solana network,
+> rent is paid to store data on the blockchain.
+
+Currently, the base Solana transaction fee is set at a static value of
+[5k lamports per signature](#). On top of this base fee, any additional
+[prioritization fees](#prioritization-fee) can be added.
+
+### Why pay transaction fees?
+
+Transaction fees offer many benefits in the Solana
+[economic design](#basic-economic-design) described below, mainly:
+
+- they provide compensation to the validator network for the CPU/GPU resources
+  necessary to process transactions,
+- reduce network spam by introducing real cost to transactions,
+- and provide long-term economic stability to the network through a
+  protocol-captured minimum fee amount per transaction
```

Comment:
Very minor, but this is an odd combination of sentence and bullet points. I'd just do bullet points without the commas and 'and'